docs/DafnyRef/Statements.md

```diff
@@ -922,7 +922,7 @@ the same as for match expressions and is described in
 
 The selector need not be enclosed in parentheses; the sequence of cases may but need not be enclosed in braces.
 The cases need not be disjoint.
-The cases must be exhaustive, but you can use a wild variable (`_`) or an as yet unused simple identifier to indicate "match anything".
+The cases must be exhaustive, but you can use a wild variable (`_`) or a simple identifier to indicate "match anything". It is possible to shadow existing variable names.
```

Comment:
While the proposed change is a little clearer than the original, I think it would be overall better to instead direct the reader to section 9.31.2 (as the previous paragraph does) for details about binding behavior, since the proposed change is still imprecise - for example, does a simple identifier shadow constants or matching parameterless constructors?

Even better, section 9.31.2 should include examples of each of the described patterns.

Reply:
Good idea for the first one. I rephrased to add a link to that section. Adding examples will be a bit more complicated since it involves also describing all the cases of shadowing, etc, so I suggest we hold off for now.

Reply:
That sounds good to me, thanks.